markets/bfp-market/contracts/storage/PerpMarket.sol

```diff
@@ -185,6 +185,21 @@ library PerpMarket {
             self.depositedCollateral[addresses.sUsd];
     }
 
+    /// @dev Returns the market's required minimum backing credit in USD.
+    function getMinimumCreditWithTradeSize(
+        PerpMarket.Data storage self,
+        PerpMarketConfiguration.Data storage marketConfig,
+        uint256 price,
+        AddressRegistry.Data memory addresses,
```

Comment:
Minor: You'll notice that `addresses` is also the last arg in all internal functions. Can we keep this consistent with that please.

Reply:
Yes, of course, I haven't noticed that but I'll update it.